src/gui/mainwindow.h

```diff
@@ -215,6 +216,8 @@ private slots:
     void displaySearchTab(bool enable);
     void createTorrentTriggered(const QString &path = {});
     void showStatusBar(bool show);
+    bool isFiltersSidebarVisible();
```

Comment:
`bool isFiltersSidebarVisible() const;`

Reply:
https://travis-ci.org/qbittorrent/qBittorrent/jobs/567771030#L1163
You forgot to change it in .cpp file too.

Reply:
Whoops, updated the cpp